src/mz/src/main.rs

```diff
@@ -62,8 +67,8 @@ enum Commands {
         #[clap(short, long)]
         interactive: bool,
     },
-    /// Show commands for interaction with the region
-    Regions(Regions),
+    /// Show commands to interact with regions
+    Region(CRegion),
```

Comment:
Can this be an inline struct instead, like the `Shell` below? I think that saves us from adding too many names here.

Reply:
I revisited [clap docs](https://docs.rs/clap/latest/clap/_derive/index.html#terminology): `struct (arguments) or enum (subcommands).`

Unfortunately, enums can't be anonymous in rust (and inline), which makes me unable to make it the other way :(.

Reply:
I don't know how but I managed to add a struct as a subcommand using derives.

Reply:
Hah, we do the thing I suggested on line [73](https://github.com/MaterializeInc/materialize/pull/14970/files/d59a9a63aa47b64288eba7aa178f1e911e2adc1f#diff-bea12c15a2b2c74c54585d6fa33e63d2457564ac96bd634bee6b3af4a57f6aeeR73) (:

Reply:
Exactly!